### PR TITLE
Get all stubbed data.

### PR DESCRIPTION
**Pull request by hgoody88**
Essentially I have used an interface for the repo and then made a dummy repo implement that interface to return data. I did this to hopefully make transitioning to a repo connected to a real db as easy as possible, unsure if that's really how to do it though?

Also if you could check out the test file as how I have setup the Controller is a bit clunky.
The repo is MOCKED.
The service has mocks INJECTED (making it a real service just with injections real).
I then passed this service as an ARGUMENT to the controller as I couldn't find a way to alert java/spring etc that this was an injectable service.

### CODE REVIEW

File: src/main/java/com/scottlogic/practice/mealscheduler/Services/MealServiceConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MealServiceConfiguration {
+    @Bean
+    MealService createMealService(MealRepo mealRepo) {
+        return new MealService(mealRepo);
+    }
+}
```
Comment (hgoody88) [inline src/main/java/com/scottlogic/practice/mealscheduler/Services/MealServiceConfiguration.java]: Is this not exactly what adding `@Service` does? I know you said you thought it was better for the future but for now I don't understand what benefit this adds except another file to look after?

File: src/main/java/com/scottlogic/practice/mealscheduler/Services/MealService.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.scottlogic.practice.mealscheduler.Services;
+
+import com.scottlogic.practice.mealscheduler.Models.Meal;
+import com.scottlogic.practice.mealscheduler.Repositories.MealRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
```
Comment (hgoody88) [inline src/main/java/com/scottlogic/practice/mealscheduler/Services/MealService.java:10]: If we're going to extract the beans to a config do we still need this?

File: src/main/java/com/scottlogic/practice/mealscheduler/Services/MealServiceConfiguration.java (hunk 1 above)
Comment (asulkowski-scott-logic) [reply src/main/java/com/scottlogic/practice/mealscheduler/Services/MealServiceConfiguration.java]: I found it useful in the past to have a possibility to create at this place some more objects that the constructor may require - either different implementations of an interface depending on settings etc. As I mentioned, it is not a 'must'. Re-factor if you prefer.

File: src/main/java/com/scottlogic/practice/mealscheduler/Services/MealService.java (hunk 2 above)
Comment (asulkowski-scott-logic) [reply src/main/java/com/scottlogic/practice/mealscheduler/Services/MealService.java:10]: Yes, we should go this or that way, not both :)

**Review by asulkowski-scott-logic — APPROVED** [review]

**Pull request merged**